docs/requirements-docs.txt

```diff
@@ -1,10 +1,11 @@
 elasticsearch>=7.7
-pandas>=1.2.0
-matplotlib
+pandas==1.5.0
```

Comment:
Please adjust the README too - that says 1.3+

Reply:
Fixed in [b07d145](https://github.com/elastic/eland/pull/512/commits/b07d14551ca1c8fc531563abfb1f12b3761d7f5c)